ui/admin/app/routes/_auth.chat-threads._index.tsx

```diff
@@ -58,7 +58,7 @@ export async function clientLoader({
 	return query ?? {};
 }
 
-export default function TaskRuns() {
+export default function ChatThreads() {
```

Comment:
why does this name keep bouncing back and forth between ChatThreads and TaskRuns? Which is it for?